qa/rpc-tests/p2p-versionbits-warning.py

```diff
@@ -155,13 +165,13 @@ def run_test(self):
             pass
         self.nodes[0] = start_node(0, self.options.tmpdir, ["-debug", "-logtimemicros=1", "-alertnotify=echo %s >> \"" + self.alert_filename + "\""])
 
-        # Connecting one block should be enough to generate an error.
+        # Since there are no unknown versionbits exceeding threshold in last period,
+        # no error will be generated.
         self.nodes[0].generate(1)
```

Comment:
The removed final check was for an error message that an unknown bit had been activated. This determination is no longer possible when unknown bits have unknown activation thresholds, hence the final test was obsolete.